src/main/main.cpp

```diff
@@ -48,8 +54,13 @@ int main(int argc, char *argv[])
 
     QGuiApplication app(argc, argv);
 
+    #ifdef Q_OS_MACOS
+    MacOsEventListener evListener;
+    initMacOsEventListener(&evListener);
```

Comment:
Can you intend these lines?
I'm not sure what's the recommended convention for this (CC @plfiorini) but I intended all compiler directives. So to be consistent, we need to either intend all or don't intend them.